### PR TITLE
3.0.0-Parallax

### PR DESCRIPTION
**Pull request by CloudPower97**
# Description

This Pull Request is for #602.

Dropping __jQuery__ support.

Adding new options (as per the [documentation](https://materializecss.com/parallax.html#options)):

- responsiveThreshold

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Using the new mocker;
Also added a new test.

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

**Comment by alextrastero** [comment]
Fixes #602

File: src/Parallax.js [hunk 1]
```diff
@@ -4,14 +4,26 @@ import cx from 'classnames';
 
 class Parallax extends Component {
   componentDidMount() {
-    $('.parallax').parallax();
+    if (M) {
```
Comment (alextrastero) [inline src/Parallax.js]: again let's keep this consistent :)

**Comment by CloudPower97** [comment]
@alextrastero ping :smile:

**Comment by alextrastero** [comment]
Very good

**Pull request merged**